.github/workflows/main.yml

```diff
@@ -40,12 +40,14 @@ jobs:
     runs-on: macos-10.15
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
       - uses: actions/setup-python@v1
         with:
           python-version: '3.7'
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip meson
+          gem install ronn
```

Comment:
*maintaining* them is not that bad since the tools don't usually change much. *Writing* them initially is terrible though but I could just convert them from `ronn` locally and check in the result instead? What you'd lose there though is the ability to link to the man page in an issue and have the users not scream at the sight of pure roff :)

If you're happy with [forking shells in `meson.build`](https://github.com/swaywm/sway/blob/master/meson.build#L125) we could use `scdoc` but honestly, I'd prefer roff instead [1]


[1] I'm not sure scdoc works under MacOS/Windows and our chances of getting patches accepted are slim

Reply:
> Writing them initially is terrible though but I could just convert them from ronn locally and check in the result instead?

Let's do that (assuming `ronn` output is reasonable...).  I don't have any experience with maintaing man pages so my inclination is to start with the basics and seeing how it works out.

> What you'd lose there though is the ability to link to the man page in an issue and have the users not scream at the sight of pure roff :)

Reply:
I've done that now in the latest push but as a follow-up commit. The source files changed enough over the commits (for the getopt long removal, etc.) that having to rebase this into the original commits was too much of a pain for very little gain.